libs/ledgerjs/packages/hw-transport-node-hid-singleton/src/TransportNodeHid.ts

```diff
@@ -143,72 +177,113 @@ export default class TransportNodeHidSingleton extends TransportNodeHidNoEvents
    *
    * Legacy: `_descriptor` is needed to follow the Transport definition
    */
-  static open(
+  static async open(
```

Comment:
Oh yeah i went a bit fast on those, i saw the `This may be converted to an async function ts(80006)` hint and went ahead. And yeah the `Promise.resolve().then` seems wrong, thank you !